msgpack/unpack.scm

```diff
@@ -47,19 +47,22 @@
               #:select (ext)))
 
 ;; ----------------------------------------------------------------------------
-(define (unpack-from in)
+(define* (unpack-from in #:key strdec)
```

Comment:
Keyword arguments can have a default value. Then we don't need to differentiate later between `#f` and a function.

```suggestion
(define* (unpack-from in #:key (strdec utf8->string))
```